app/views/smart_answers/result.html.erb

```diff
@@ -15,16 +15,13 @@
 <div class="govuk-grid-row">
   <div id="result-info" class="govuk-grid-column-two-thirds outcome">
     <%= render 'smart_answers/debug' %>
-    <%= render "govuk_publishing_components/components/title", {
-      title: outcome.heading_title
-    } %>
-
     <div class="govuk-!-margin-bottom-6 result-body">
       <% if outcome.title.present? %>
-        <%= render "govuk_publishing_components/components/heading", {
-          text: outcome.title,
-          margin_bottom: 6
-        } unless outcome.title_as_heading? %>
+        <%= render "govuk_publishing_components/components/title", {
+        context: outcome.heading_title,
+        title: outcome.title,
+        margin_bottom: 5
+      } unless outcome.title_as_heading? %>
```

Comment:
This assumes all smart answers have a title specified on a outcome page - which isn't the case. E.g. 
The (part-year-profit-tax-credits flow)[https://www.gov.uk/part-year-profit-tax-credits/y/2012-10-01/0000-05-05/yes/no/2012-08-01/2013-04-04/10000.0] doesn't have a title and would need content update first.

Reply:
https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/part-year-profit-tax-credits/outcomes/result.erb

Reply:
Ace - thanks @theseanything for the insight!  Yeah, there's a few things not quite right with this solution (including this)  I've raised this as a draft right now as due to for a quick chat about the ticket.

Reply:
👋🏼 @theseanything still in draft, however it's at the stage where a cursory glance might be helpful.